pkg/sdk/resource_monitors.go

```diff
@@ -97,7 +98,13 @@ func (row *resourceMonitorRow) convert() (*ResourceMonitor, error) {
 		resourceMonitor.Frequency = *frequency
 	}
 	if row.StartTime.Valid {
-		resourceMonitor.StartTime = row.StartTime.String
+		const YYMMDDhhmm = "2006-01-02 15:04"
+		t, err := time.Parse(time.RFC3339, row.StartTime.String)
+		if err != nil {
+			return nil, err
+		}
+		localTime := t.Local()
+		resourceMonitor.StartTime = localTime.Format(YYMMDDhhmm)
 	}
 	if row.EndTime.Valid {
```

Comment:
The issue was also mentioning the end timestamp, so shouldn't this also change?

Reply:
The issue also mentions the `notify_users` param. Did you look into that?

Reply:
Good point - I'll add